CodingStyle

```diff
@@ -79,7 +79,7 @@ by section.
   header is:
 
 // -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
-// vim: ts=8 sw=2 smarttab
+// vim: ts=8 sw=2 smarttab ft=cpp
```

Comment:
@liewegas any objection to this change in CodingStyle? the intent is to identify .h files as c++ instead of c, to match the 'mode:C++' for emacs

Reply:
Sounds like a great idea to me!